lightning-invoice/src/lib.rs

```diff
@@ -506,6 +510,12 @@ impl<D: tb::Bool, H: tb::Bool, T: tb::Bool> InvoiceBuilder<D, H, T> {
 		}
 		self
 	}
+
+	/// Adds feature bits.
```

Comment:
This should have more details - "Adds a features field which indicates the set of supported protocol extensions which the invoice generator supports".